### PR TITLE
Prac 06 feedback

### PR DESCRIPTION
**Pull request by taitymrobinson**
Could you please review the overall formatting of code, use of testing and readability? @phoebe365

### CODE REVIEW

File: prac_06/guitar_test.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+"""
+CP1404 - PRAC 06
+"""
+from prac_06.guitar import Guitar
+
+gibson = Guitar("Gibson L-5 CES", 1922, 16035.40)
+bass = Guitar("Bass L-5 CES", 1999, 134563.40)
+
+print(f"{gibson.name} get_age() - Expected 102. Got {gibson.get_age()}")
```
Comment (phoebe365) [inline prac_06/guitar_test.py:9]: I like the way you set up your testing to print a string, in one line. Defiantly efficient and readable.

File: prac_06/guitar.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+
+    def get_age(self):
+        """Gets the age of the guitar."""
+        return CURRENT_YEAR - self.year
+
+
+    def is_vintage(self):
+        """Checks if the guitar is vintage."""
+        return self.get_age() >= VINTAGE_AGE
```
Comment (phoebe365) [inline prac_06/guitar.py:30]: I like how you used constants for vintage age and current year, makes the code easier to read and understand.

File: prac_06/guitars.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        print(f"{new_guitar.__str__()} added.")
+        name = input("Name: ")
+
+    guitars.append(Guitar("Gibson L-5 CES", 1922, 16035.40))
+    guitars.append(Guitar("Line 6 JTV-59", 2010, 1512.9))
+
+    print("-" * 20) # Used to improve readability
+
+    for i, guitar in enumerate(guitars, 1):
+        vintage_string = "vintage" if guitar.is_vintage() else "not vintage"
```
Comment (phoebe365) [inline prac_06/guitars.py:25]: Doing the vintage check in one line is efficient and works well, however I'm just wondering if Lindsay would prefer if we practice our if else layouts.

File: prac_06/REFLECTION.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+experience, compared to in previous classes where I was not an active participant.
+
+## What do you need to stop or change in your work for this subject overall so far?
+
+I need to stop relying on 'last-minute' urgency to complete assignment tasks as it will allow me to create better
+submissions and overall improve my learning journey.
+
+## What is one request you have or change you suggest for this subject and/or your IT degree overall?
+
+I would like to see an increase of the 'quiz' aspects within lecture videos, and an increase in example exam
```
Comment (phoebe365) [inline prac_06/REFLECTION.md:28]: love this idea

File: prac_06/car.py [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+            distance = self.fuel
+            self.fuel = 0
+        else:
+            self.fuel -= distance
+        self._odometer += distance
+        return distance
+
+    def __str__(self):
+        """Display the data for car object"""
+        return f"{self.name}, fuel={self.fuel}, odometer={self._odometer}"
```
Comment (phoebe365) [inline prac_06/car.py:34]: You're layout looks great, formatted nicely

**Review by phoebe365 — COMMENTED** [review]
All your formatting looks good, readability is nice -  comments are very informative, and the testing looks good as well. Well done.

**Comment by taitymrobinson** [comment]
> All your formatting looks good, readability is nice - comments are very informative, and the testing looks good as well. Well done.

Thanks for taking the time to review!!

**Pull request merged**